content/tutorials/wallets.md

```diff
@@ -1,52 +1,66 @@
 ---
-title: Wallets for Ocean Tokens
-description: If you want to use the Ocean Network, you need Ocean Tokens (typical ERC-20 tokens), and to get Ocean Tokens, you need to get a wallet that can hold Ocean Tokens.
+Title: Wallets for Ocean Tokens
```

Comment:
this will break the implementation, please don't change the spelling of the keys (`title`, `description`) in any way, deviating from the [Markdown File Requirements](https://github.com/oceanprotocol/docs#markdown-file-requirements)

Reply:
Awesome, I think this was what was causing it